resources/NetworkRestSocket.js

```diff
@@ -142,6 +157,7 @@ class NetworkRestSocket extends NetworkRest {
                }
                // Success!
                else {
+                  performance.measure(route);
                   resolve(data);
```

Comment:
Question:  so if we update our response `data` to possibly have `data.performance` could we also fold that server side performance into this current `route` set of measurements?

What if our data structure was something like:
```
{
	data:{ data returned from server },
	performance: {
		jobID: "server side JobID",
		measurements: {
			key1 : measurement1,
			key2 : measurement2,
			...
			keyN : measurementN
		}
	}
}
```

Reply:
So yes it's possible. One option is to add it as custom data, so the format can be whatever we want. But if we want it to look like something sentry tracked, we would want start and end times, ideally in epoch time (seconds).

Reply:
So, would it have to look something like:
```
{
	data:{ data returned from server },
	performance: {
		jobID: "server side JobID",
		measurements: {
			key1 : { start: epochTime1, duration: measurement1 },
			key2 : { start: epochTime2, duration: measurement2 },
			...
			keyN : { start: epochTimeN, duration: measurementN }
		}
	}
}
```
Would this then allow us to see the server measurements displayed in line with the rest of the client tracked route measurements?

What if we close out this PR and we can create another task for us to enhance this in the future?

Reply:
Yes it would display in line. Opened #418